documents/0_Introduction/ja-JP/Introduction.md

```diff
@@ -0,0 +1,24 @@
+# �����ꗗ
+### �T�v�E�Љ�
+- Introduction: .NET�p�A�v���P�[�V�����E�t���[�����[�N Open����  
+http://www.slideshare.net/nishi_74322014/introduction-34918790
+- Open����T�v�� 01-72  
+http://www.slideshare.net/daisukenishino/open-01-50-40474003
+- Open����@�\�Љ� 01-72  
+http://www.slideshare.net/daisukenishino/open-0150
+- Open������[�h�}�b�v v01-72�����[�X��  
+http://www.slideshare.net/daisukenishino/open-v0172
+- Open���� �Љ�C���t���V�X�e�� �A�[�L�e�N�`���̒��  
+http://www.slideshare.net/daisukenishino/open-47272843
+
+### �Z�~�i�[��
+- �w���ꂩ���.NET�A�v���P�[�V�����J���x�Z�~�i�[  
+.NET�p�A�v���P�[�V���� �t���[�����[�N Open ���� �T��  
+http://www.slideshare.net/daisukenishino/open-36092863
+- .NET�p�A�v���P�[�V���� �t���[�����[�N �gOpen ����h�̃I�[�v���\�[�X���ɂ���  
+http://www.slideshare.net/daisukenishino/openoss
+- Open ���� @ �I�[�v���\�[�X�J���t�@�����X 2015 Nagoya  
+http://www.slideshare.net/daisukenishino/open-2015-nagoya
+- �I�[�v���A�[�L�e�N�`���C�I�[�v���\�[�X�\�t�g�E�F�A�����  
+�W�����t���[�����[�N��g�p�����v���W�F�N�g�}�l�W�����g  
+http://www.slideshare.net/daisukenishino/ss-66717458
```

Comment:
@daisukenishino 

This web page is not displayed in case of using other browsers, such as IE, chrome.
So, I think that this is Slideshare problem, not firefox problem.

Is this web page displayed normally in @daisukenishino 's computer?
In my computer, when revising URL as following, this web page is displayed normally.
```txt
http://www.slideshare.net/daisukenishino/ss-66717458?qid=704631ce-e91f-4ba2-9639-9f604618e225
```

Please confirm.